### PR TITLE
deployment: update the README

### PR DESCRIPTION
**Pull request by hagen1778**
Signed-off-by: hagen1778 <roman@victoriametrics.com>

### CODE REVIEW

File: deployment/docker/README.md [hunk 1]
```diff
@@ -36,51 +36,49 @@ VictoriaMetrics will be accessible on the following ports:
...
 use link [http://localhost:8481/select/0/prometheus/vmui](http://localhost:8481/select/0/prometheus/vmui).
 
 ## vmagent
 
-vmagent is used for scraping and pushing timeseries to
-VictoriaMetrics instance. It accepts Prometheus-compatible
-configuration `prometheus.yml` with listed targets for scraping.
+vmagent is used for scraping and pushing time series to VictoriaMetrics instance. 
+It accepts Prometheus-compatible configuration [prometheus.yml](https://github.com/VictoriaMetrics/VictoriaMetrics/blob/master/deployment/docker/prometheus.yml)
+with listed targets for scraping.
```
Comment (tenmozes) [inline deployment/docker/README.md:74]: should we mentioned that it supports only limited configuration from this file?

File: deployment/docker/README.md [hunk 2]
```diff
@@ -36,51 +36,49 @@ VictoriaMetrics will be accessible on the following ports:
...
 
 To access [vmui](https://docs.victoriametrics.com/Single-server-VictoriaMetrics.html#vmui) 
 use link [http://localhost:8481/select/0/prometheus/vmui](http://localhost:8481/select/0/prometheus/vmui).
 
 ## vmagent
 
-vmagent is used for scraping and pushing timeseries to
-VictoriaMetrics instance. It accepts Prometheus-compatible
-configuration `prometheus.yml` with listed targets for scraping.
+vmagent is used for scraping and pushing time series to VictoriaMetrics instance.
```
[the remark this answers is not in the document]
Comment (valyala) [reply deployment/docker/README.md:72]: This isn't the main docs for VictoriaMetrics - these are supplementary docs for docker-compose, so I think there is no sense in bloating these docs.

File: deployment/docker/README.md (hunk 1 above)
Comment (valyala) [reply deployment/docker/README.md:74]: This isn't the main docs for VictoriaMetrics - these are supplementary docs for docker-compose, so I think there is no sense in bloating these docs.

**Review by valyala — APPROVED** [review]

**Comment by valyala** [comment]
@hagen1778 , thanks for the pull request!

**Pull request merged**